hw20/roles/hw201/tasks/ubuntu.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: "Install programs on Ubuntu"
```

Comment:
Add `when` condition for ubuntu to be more flexible

Reply:
complete